use_case_examples/hybrid_model/serve.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+uname_str=$(uname)
+echo "${uname_str}"
+if [[ $uname_str != "Darwin" ]]; then
+  echo "Not Darwin"
+  # tune the cpu-list according to the resources you want to allocate to it
+  PATH_TO_MODELS="compiled_models" PORT=8000 taskset --cpu-list 0-12 python serve_model.py
+  # No-limit
+  # PATH_TO_MODELS="compiled_models" PORT=8000 python serve_model.py
```

Comment:
to remove ?

Reply:
I'd rather keep it here so that people can easily un-comment it if they want

Reply:
could become an ENV var to enable or something like that then, but sure

Reply:
Could be indeed but I'd rather have people modifying this file, we don't all have the same number of CPU cores for example

Reply:
got it !